qa/tasks/mgr/dashboard/test_rgw.py

```diff
@@ -67,7 +66,6 @@ def get_rgw_user(self, uid, stats=True):
         return self._get('/api/rgw/user/{}?stats={}'.format(uid, stats))
 
 
-@skip_unless_dashboard_pr
```

Comment:
But this will continue failing no matter how many retries happen, since the root issue has not been fixed.